README.md

```diff
@@ -1 +1,163 @@
-# atracker
\ No newline at end of file
+# IBM Cloud Activity Tracker with COS - Terraform Module
+<br/>
+
+Observability services - gives you the visibility into the performance and health of your resources on IBM Cloud. You can use these services to troubleshoot your apps and services, identify threats, detect performance issues, trigger alerts and more.
+
+This module is used to create observabillty service with Activity Tracking with Cloud Object storage solution on IBM Cloud Platform.
+
+Atracker module provision:
+- Atracker
+
+# Compatibility
+<br/>
+This module is meant to use with Terraform 0.13 (and higher)
+
+# Example Usage
+<br/>
+
+Examples found in [examples folder](https://github.ibm.com/slz-v2-poc/terraform-atracker-module/tree/master/Examples).
+
+```
+provider "ibm" {
+}
+
+data "ibm_resource_group" "cos_group" {
+  name = var.resource_group
+}
+
+module "cos" {
+  source  = "terraform-ibm-modules/cos/ibm//modules/instance"
+  version = "1.4.1"
+
+  service_name      = var.prefix != null ? "${var.prefix}-${var.service_name}" : var.service_name
+  plan              = var.plan
+  region            = var.region
+  resource_group_id = data.ibm_resource_group.cos_group.id
+  parameters        = var.parameters
+
+  bind_resource_key = var.bind_resource_key
+  resource_key_name = var.resource_key_name
+  key_parameters    = var.key_parameters
+  role              = var.role
+}
+
+module "cos_bucket" {
+  source  = "terraform-ibm-modules/cos/ibm//modules/bucket"
+  version = "1.4.1"
+
+  bucket_name     = var.prefix != null ? "${var.prefix}-${var.bucket_name}" : var.bucket_name
+  cos_instance_id = module.cos.cos_instance_id
+  location        = var.location
+  storage_class   = var.storage_class
+  force_delete    = var.force_delete
+  endpoint_type   = var.endpoint_type
+  kms_key_crn     = var.kms_key_crn
+  expire_rules    = var.expire_rules
+}
+
+module "atracker" {
```

Comment:
Why is it pointing to cos instance module

it should be pointing to root module as you have done in examples/atracker/main.tf

module "atracker" {
  source = "../../"

  resource_group = var.resource_group
  bucket_name    = var.bucket_name
  location       = var.location
  target_crn     = module.cos.cos_instance_id
  api_key        = module.cos.cos_key_credentials.apikey
}

Reply:
https://github.com/terraform-ibm-modules/terraform-ibm-atracker/pull/2/commits/091eafa2284351400f813a312e702153ba68f21a